directory_forms_api_client/client.py

```diff
@@ -14,10 +14,10 @@ def ping(self):
         return self.get(url=self.endpoints['ping'])
 
     def submit_generic(self, data):
-        return self.post(url=self.endpoints['generic-submig'], data=data)
+        return self.post(url=self.endpoints['generic-submit'], data=data)
```

Comment:
yeah, just a placeholder

at the very least, the endpoint should not be named simply "submit", because that will not scale well if we add a more specialist submit endpoint.

Reply:
let's make it RESTful - the resource here is form submission / result / document - can be named accordingly